### DISCUSSION TITLE
Client cannot see certain servers

### DISCUSSION THREAD

**Discussion by insomniux** [jamulussoftware/jamulus#1170] [Technical Support & Help] [answered]
Hi, I host one public Jamulus server and two private servers. All servers run on a virtual host at an ISP in the Netherlands. All three servers can be seen an used by several Jamulus clients all over the country. 

However we have (at least) one jamulus client who cannot see my servers (neither the private nor the public), but she can connect to a public server in Germany. Ive been able to have an audio conversation with her through this German server. I'm using the generic jamulus ports (22124) What could be the reason that she cannot see nearby servers but can connect through a remote server (although not useful for rehearsal due to the high latency)?

**Comment by ann0see** [comment]
I assume that's due to network related issues. Just to be clear: can this client connect to all server via IP?

**Comment by gene96817** [comment, marked as answer]
Here is how I would triage the issue:
-- the server is operating "normally".... other clients can access the server using normal means
-- the client in question is operating normally .... it can connect to other servers
-- the desired server is not displayed --> implies the client-server ping failed
                                                                  --> client could ping other servers
                                                                  --> the server responded to pings from other clients
                                                                  --> it is likely the ping was lost either going or returning from the server
-- test making a connection using IP address
                                                                   --> If succeeds, then ping was lost in network traffic
                                                                    --> if failed, then there is a routing problem between the client's network and the server's network.
-- confirm routing problem with a traceroute and report to ISP

**Reply by insomniux** [reply]
Thanks gene96817 and ann0see for your hints. The client cannot connect using the IP address of the private server and cannot see my public server in the list. Following  gene96817's triage we end up at a routing problem at the ISP, as the client can see some other public servers. And it seems not ping-related as also using the IP address of the private-server fails. We'll first try to traceroute the connections and relay to the ISP.
